plugins/code-coverage-backend/README.md

```diff
@@ -2,6 +2,63 @@
 
 This is the backend part of the `code-coverage` plugin. It takes care of processing various coverage formats and standardizing them into a single json format, used by the frontend.
 
+## Installation
+
+`yarn workspace backend add @backstage/plugin-code-coverage-backend`
+
+Next you will need to add code to setup the plugins router, in `packages/backend/src/plugins/codecoverage.ts` add the following:
+
+```typescript
+import { createRouter } from '@backstage/plugin-code-coverage-backend';
+import { Router } from 'express';
+import { PluginEnvironment } from '../types';
+
+export default async function createPlugin(
+  env: PluginEnvironment,
+): Promise<Router> {
+  return await createRouter({
+    config: env.config,
+    discovery: env.discovery,
+    database: env.database,
+    urlReader: env.reader,
+    logger: env.logger,
+  });
+}
+```
+
+Finally we need to load the plugin in `packages/backend/src/index.ts`, make the following edits:
+
+```typescript
```

Comment:
Hmm, now the diff isn't showing properly. I think the code block should be `diff` and not `typescript`. My suggestion was just to add a comment. Sorry if that was confusing. Basically I've ran into some issues with people on Discord setting up various plugins so I tend to include more details as it makes things easier. The Azure DevOps plugin is a good example of how I might do this: https://github.com/backstage/backstage/tree/master/plugins/azure-devops-backend#up-and-running. Look at step 3

Reply:
Updated to show the diff